namer/namer.h

```diff
@@ -5,10 +5,18 @@
 
 namespace sorbet::namer {
 
+class NamerCtx final {
```

Comment:
I'm not happy with this solution but was running into a lot of issues with the templating from namer and forward declarations.